.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Description and relevant Jira numbers
+
+## Pull Request checklist:
+
+- [ ] [WIP] tag removed from PR title
+- [ ] PR has an understandable description
+
+## Git feature branch checklist:
+
+- [ ] branch rebased against the latest develop
+- [ ] commits contain relevant JIRA ticket number
+- [ ] commits are squashed
+
+## Sign off process checklist:
+
+- [ ] Code been tested manually
+- [ ] Tests are passing
+- [ ] PR link added to JIRA
+- [ ] Tests are passing
+- [ ] Reviewers selected in Github
+
+- [ ] Screenshot (if applicable)
```

Comment:
can we specify this needs to be from the physical device?

Reply:
done